src/why/index.tsx

```diff
@@ -20,6 +20,14 @@ const StyledWhy = styled(Why)`
     background-color: ${color.lightBackground};
   }
 
+  :focus:not(.focus-visible) {
+    outline: none;
+  }
+
+  .focus-visible {
+    outline: 1;
+  }
```

Comment:
@simonrelet by using the provider value, if the user is focused on this component by using keyboard he will see this outline style.

Reply:
Ok I see, thanks.

Reply:
It looks like the selector `.focus-visible` (line 27) can be removed as everything is handled by the selector `:focus:not(.focus-visible)` (line 23)

Reply:
Ah yes you right, it's not required now.